.claude/PRPs/issues/issue-978.md

```
@@ -0,0 +1,538 @@
+# Investigation: One-command web UI install via `archon serve`
+
+**Issue**: #978 (https://github.com/coleam00/Archon/issues/978)
+**Type**: ENHANCEMENT
+**Investigated**: 2026-04-09T12:00:00Z
+
+### Assessment
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)